TestSuite/integration.py

```diff
@@ -56,13 +56,19 @@ def build(
         if image is not None:
             self.image.write_bytes(image)
 
-    def create_default_integration(self):
+    def create_default_integration(self, name: str = '', commands: str = ''):
         default_integration_dir = 'assets/default_integration'
 
         with open(suite_join_path(default_integration_dir, 'sample.py')) as code_file:
             code = str(code_file.read())
         with open(suite_join_path(default_integration_dir, 'sample.yml')) as yml_file:
             yml = yaml.safe_load(yml_file)
+            if name:
+                yml['name'] = name
+                yml['commonfields']['id'] = name
+            if commands:
+                for command in commands:
```

Comment:
`commands` is a string according to the annotation above.
We should either accept it as a comma-separated value or as a list.